pom.xml

```
@@ -38,6 +38,7 @@
     <spring-boot.version>2.4.5</spring-boot.version>
     <spring-cloud.version>2020.0.2</spring-cloud.version>
     <spring.version>5.3.6</spring.version>
+    <spring.security.version>5.3.6.RELEASE</spring.security.version>
```

```suggestion
    <spring.security.version>5.4.6</spring.security.version>
```

According to https://docs.spring.io/spring-boot/docs/2.4.5/reference/html/appendix-dependency-versions.html it should be 5.4.6